ykrt/src/mt.rs

```diff
@@ -867,6 +877,12 @@ impl MT {
         }
     }
 
+    /// Inform this `MT` instance that `deopt` has occurred: this updates the stack of
+    /// [MTThreadState]s.
+    pub(crate) fn deopt(self: &Arc<Self>) {
+        MTThread::with(|mtt| mtt.pop_tstate());
```

Comment:
shall we assert that the top of the stack is `Executing` here for sanity?

Reply:
Fixed in 116ba0371f9c1f5a03b5b83ee986907ca4065a0b.

Reply:
Thanks!